int32 Vx
float32 Vz
float32 AccX
float32 AccY
float32 AccZ
float32 GyrX
float32 GyrY
float32 GyrZ
int16 Voltage
int16 State
int16 Light12
int16 Light34